.travis.yml

```diff
@@ -2,4 +2,6 @@ language: python
 python:
   - "2.7"
 install: "pip install -r requirements.txt"
-script: python test.py
+script:
+- python test.py
+- python setup.py install
```

Comment:
Can you add `- python setup.py build` before the `install` step, then this is good to merge.